coalib/results/AbsolutePosition.py

```diff
@@ -27,6 +27,90 @@ def __init__(self,
     def position(self):
         return self._position
 
+    def __le__(self, other):
+        """
+        Test whether ``self`` is behind or equals the other
+        ``AbsolutePosition``.
+
+        :param other: ``AbsolutePosition`` to compare with.
+        :return:      Whether this ``AbsolutePosition`` is behind the other
+                      one or the same.
```

Comment:
tell precisely that it returns ``true`` when .... and raise a TypeError when ...